nginx-app/recipes/configure.rb

```diff
@@ -63,6 +63,7 @@
       :doc_root           => ::EasyBib::Config.get_appdata(node, application, 'doc_root_dir'),
       :app_dir            => ::EasyBib::Config.get_appdata(node, application, 'app_dir')
     )
+    notifies :run, 'execute[configtest]', :immediately
```

Comment:
@baccenfutter I think that's one place. The other is in `easybib_nginx`. 

If you grep for nginx and template, you should find this recipe **and** the LWRP:

```
$ fgrep -r 'template ' *|grep 'nginx'